cmd/notary/tuf.go

```diff
@@ -428,19 +428,10 @@ func importRootKey(cmd *cobra.Command, rootKey string, nRepo *notaryclient.Notar
 	var rootKeyList []string
 
 	if rootKey != "" {
-		privKey, err := readKey(data.CanonicalRootRole, rootKey, retriever)
-		if err != nil {
-			return nil, err
-		}
-		// add root key to repo
-		err = nRepo.CryptoService.AddKey(data.CanonicalRootRole, "", privKey)
-		if err != nil {
-			return nil, fmt.Errorf("Error importing key: %v", err)
-		}
-		rootKeyList = []string{privKey.ID()}
-	} else {
-		rootKeyList = nRepo.CryptoService.ListKeys(data.CanonicalRootRole)
+		cmd.Printf("import root key: ")
```

Comment:
Or this should be changed to `cmd.Printf(Using root key from command input: %s\n, rootKeyID)`, and then the test should be updated accordingly